inginious/frontend/submission_manager.py

```diff
@@ -364,15 +364,21 @@ def get_feedback_from_submission(self, submission, only_feedback=False, show_eve
         if "text" in submission:
             submission["text"] = ParsableText(submission["text"], submission["response_type"], show_everything, translation).parse()
         if "problems" in submission:
-            for problem in submission["problems"]:
-                if isinstance(submission["problems"][problem], str):  # fallback for old-style submissions
-                    submission["problems"][problem] = (submission.get('result', 'crash'), ParsableText(submission["problems"][problem],
-                                                                                                       submission["response_type"],
```

Comment:
It feels like we are silently dropping support for "response_type" in subproblem-specific feedback here. Is it expected?

Reply:
The feature is deprecated since v0.4 but they are still people using it through the undocumented YAML fields. So yes maybe this should be kept for now.